test/ao/sparsity/test_pruner.py

```diff
@@ -150,6 +171,25 @@ def test_prepare_linear(self):
             assert type(module.parametrizations.weight[0]) == PruningParametrization
         assert model(x).shape == (128, 16)
 
+    def test_prepare_linear_cuda(self):
```

Comment:
Do we need separate tests for all the `cuda` cases or can we parametrize the existing one to work on both devices?

Reply:
I agree -- we can wrap the test in a separate function:

```
def _test_prepare_linear_on_device(self, device):
    print(f'Testing on {device.type}')
    # All of your testing code here

def test_prepare_linear(self):
    self._test_prepare_linear_on_device(torch.device('cpu'))
    if torch.cuda.is_available():
        self._test_prepare_linear_on_device(torch.device('cuda'))
```